### ISSUE TITLE
Magnitude has an infinite regress of precisions

### ISSUE THREAD

**Issue by uscholdm** [semanticarts/gist#980]
Currently every `Magnitude` has a precision that is also a `Magnitude`. So if I weigh myself as 80kg on my scale at home that is accurate to 500g, then the precision is the `Magnitude` that represents 500g. But according to gist, that `Magnitude` also has a precision. I cannot think what it might mean for a precision to have a precision. 

The problem is that the precision characterizes the weighing event and the accuracy of the scale, not the magnitude itself.  The magnitude itself is just a number and a unit; it's a mathematical construct used to describe the physical world.  If 10 different people weigh in at 80kg on 10 different scales, with different accuracies, there need only be a single magnitude instance for 80kg, and the 10 measuring events would each have a precision based on the scale and possibly other factors. 

Proposal:
- remove the `hasPrecision` restriction from `Magnitude`
- keep the property `hasPrecision` and give it a scope note indicating how it might be used. For example.

``` 
skos:scopeNote "the event of measuring a person's weight at the doctor's office may have an associated precision of 1 pound"^^xsd:string ; 
```

**Comment by dylan-sa** [comment]
Yes, I agree that precisions aren't inherent to magnitudes; it's probably particular measurements (like weighing events) that have precisions, as you suggest. I'd be in favor of removing the restriction.

I see some discussion of a `gist:Measurement` class here: https://github.com/semanticarts/gist/issues/18. I'm not advocating that we add the class, but may be worth considering.

**Comment by sa-bpelakh** [comment]
Agree w/ @dylan-sa , precision is an aspect of a measurement (or even a sensor, in an IoT world), and does not belong on every Magnitude. In case of use case where measurements vary widely in precision, a sub-class `PreciseMagnitude` can be defined which bakes precision into every instance.

**Comment by philblackwood** [comment]
Precision is actually precision of the measurement method, but can also be interpreted as likelihood of error in the value of the measurement. 

I think this is resolved by the interpretation given with hasPrecision in the new Units of Measure:

skos:scopeNote "A typical way to use precision is to have it represent 2 standard deviations of the distribution of measurement errors. With this convention, when the measurement method is well-calibrated (has an average error of zero), there is a 95% chance that the actual error in measurement, in either direction, is less than the precision.

(also implies the measurement method results in a normal distribution of errors)

**Comment by uscholdm** [comment]
> Precision is actually precision of the measurement method, but can also be interpreted as likelihood of error in the value of the measurement.

"Can be" is one thing, which is fine. However, that is optional. Putting a precision restriction on a magnitude seems to be incorrectly forcing that view.

**Comment by uscholdm** [comment]
Copied from https://github.com/semanticarts/gist/pull/1064

@philblackwood comments and asks: 
- Yes. In fact, a precision does have a precision, and if anybody wanted to record these they could. Are there any practical negative consequences?

To which I reply: 
- That's a good question. An alternate view is to consider what are the practical positive consequences of requiring every Magnitude to have a precision.  As others have said in the above comments, the precision rightfully belongs on the event of doing the measurement, not the outcome of the measurement.

**Comment by philblackwood** [comment]
Michael, my question was if there are any negative consequences to the fact that precision can have a precision. It seems to be a fact, even though in practice it is probably almost never done. If you think the object of hasPrecision is not a magnitude, then what should it be?

**Comment by philblackwood** [comment]
Does anybody disagree with the following scope note for hasPrecision?

A typical way to use precision is to have it represent 2 standard deviations of the distribution of measurement errors. With this convention, when the measurement method is well-calibrated (has an average error of zero), there is a 95% chance that the actual error in measurement, in either direction, is less than the precision.

**Comment by philblackwood** [comment]
"Doctors say that Nordberg has a 50-50 chance of living, though there's only a 10 percent chance of that."

**Comment by uscholdm** [comment]
> Does anybody disagree with the following scope note for hasPrecision?
> 
> A typical way to use precision is to have it represent 2 standard deviations of the distribution of measurement errors. With this convention, when the measurement method is well-calibrated (has an average error of zero), there is a 95% chance that the actual error in measurement, in either direction, is less than the precision.

This is quite technical, after reading it a few times, I'm still not entirely sure I get it.  How about something simple like:
`skos:scopeNote "the event of measuring a person's weight at the doctor's office may have an associated precision of 1 pound"^^xsd:string ; `

**Comment by kchastain703** [comment]
When I was working with scientific instrument/measurement modelling, we had the precision as a property of the instrument used to take the measurement. With the water quality sensors, they would often list the precision in the user manual, or could be found online. So, that might be another way to handle it? I do agree that having the precision on the Magnitude doesn't quite feel right to me, though.

**Comment by philblackwood** [comment]
I removed hasPrecision from the class equivalence statement for Magnitude.

The precision is a characteristic of the measurement method, and if that is not modeled it is also valid to use hasPrecision with a Magnitude.

**Comment by stevenchalem** [comment]
No longer relevant. The hasPrecision restriction was removed from gist:Magnitude's equivalentClass and the property was renamed to hasAccuracy.
